test/stripe/core_resources/file_upload_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule Stripe.FileUploadTest do
+  use Stripe.StripeCase, async: true
+
+  describe "create/2" do
+    test "creates a file" do
+      assert {:ok, %Stripe.FileUpload{}} = Stripe.FileUpload.create(%{
+        file: "@/path/to/a/file.jpg",
+        purpose: "dispute_evidence"
+      })
+      assert_stripe_requested :post, "/v1/files"
```

Comment:
@joshsmith so perhaps these are failing is b/c it is `files.stripe.com` instead of `api.stripe.com`

Reply:
This test file didn't exist before

Reply:
ah found a `request_file_upload` method in Stripe.Request